src/components/Card.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts"></script>
+
+<template>
+  <div class="card">
+    <slot>
+
+    </slot>
+  </div>
+</template>
+
+<style scoped>
+.card {
+  border: 1px solid #ccc;
```

Comment:
we should use the color from our color style file, in case we need color add them to that file and use it with the variable name instead for the code

Reply:
done